scripts/base/frameworks/telemetry/main.zeek

```diff
@@ -580,23 +481,20 @@ event run_sync_hook()
 	schedule sync_interval { run_sync_hook() };
 	}
 
-event zeek_init()
-	{
-	schedule sync_interval { run_sync_hook() };
-	}
-
 # Expose the Zeek version as Prometheus style info metric
 global version_gauge_family = Telemetry::register_gauge_family([
 	$prefix="zeek",
 	$name="version_info",
-	$unit="1",
+	$unit="",
 	$help_text="The Zeek version",
 	$labels=vector("version_number", "major", "minor", "patch", "commit",
                        "beta", "debug","version_string")
 ]);
 
 event zeek_init()
 	{
+	schedule sync_interval { run_sync_hook() };
```

Comment:
I don't. I wasn't super-involved in the design of the original code. I think it's so that there's only one timer needed for the internal event, instead of needing multiple for each `schedule` invocation.